cypress/e2e/03_dice_helper.cy.js

```diff
@@ -0,0 +1,70 @@
+import { clear_chat, cleanup_actors } from "../support/common.js";
+
+/***
+ * Rolls the specified skill within an open actor sheet
+ * @param skill_name
+ */
+function roll_skill(skill_name) {
+    cy.get('.sheet-tabs > [data-tab="items"]').click();
+    cy.get(":nth-child(2) > .items-header > .header-name").should("be.visible");
```

Comment:
Nit: Generally I try to avoid nth-childs that aren't obviously first or last, because I think they'll prove brittle to changes upstream to either the module or system. But, we can revisit that later if these ever break.